tron/utils/crontab.py

```diff
@@ -51,7 +51,15 @@ def parse(self, source):
             return None
 
         groups = [self.get_values(group) for group in self.get_groups(source)]
-        return sorted(set(itertools.chain.from_iterable(groups)))
+        groups = set(itertools.chain.from_iterable(groups))
+        has_last = False
+        if 'LAST' in groups:
```

Comment:
Why is this new logic here? Changing the python version shouldn't requires change like this, no?

Reply:
this is because python 3 no longer allows comparing numbers to strings, so i have to remove `'LAST'` from the array before passing it to `sorted()`